src/client/vertxbus.js

```diff
@@ -163,10 +167,17 @@ var vertx = vertx || {};
         }
       } else {
         // Might be a reply message
-        var handler = replyHandlers[address];
-        if (handler) {
+        var handlers = replyHandlers[address];
+        if (handlers) {
           delete replyHandlers[address];
-          handler(body, replyHandler);
+          var handler = handlers.replyHandler;
```

Comment:
Won't this mean the normal reply handler will also be called when there's a failure too?